src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -80,6 +77,35 @@ public void getOaiPmhViewInstances(String startDate, String endDate, boolean del
     }
   }
 
+  @Override
+  public void getOaiPmhViewUpdatedInstanceIds(String
+                                                startDate, String endDate, boolean deletedRecordSupport,
+                                              boolean skipSuppressedFromDiscoveryRecords, @Pattern(regexp = "[a-zA-Z]{2}") String lang, RoutingContext routingContext,
+                                              Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    log.debug("request params:", Iterables.toString(routingContext.request()
+      .params()));
+
+    Tuple params = createPostgresParams(startDate, endDate, deletedRecordSupport, skipSuppressedFromDiscoveryRecords);
+    log.debug("postgres params:", params);
+    getCommonInstanceProcessing(SQL_UPDATED_INSTANCES, params, routingContext, okapiHeaders, asyncResultHandler, vertxContext,
+      "Select from oai pmh updated instances view completed successfully");
+  }
+
+  @Override
+  public void postOaiPmhViewInstances(boolean skipSuppressedFromDiscoveryRecords, List<String> entity, RoutingContext routingContext, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
Please apply standard line continuation and 2 space indentation

Reply:
Done